README.md

```diff
@@ -111,6 +113,7 @@ monolog:
             'context'     => [],
             'labels'      => [],
             'client_name' => '',
+            'is_sending_enabled' => (bool) env('IS_LOKI_LOGGING_ENABLED', true),
```

Comment:
Sorry it took me a while to get back to this.  
Please rename the parameter (and the corresponding property) to `enableSend`.
Also there seem to be some conflict in your PR